modules/storage-persistent-storage-aws-maximum-volumes.adoc

```diff
@@ -7,13 +7,9 @@
 
```

Comment:
Outside of this PR scope, but should this module heading be sentence case?

Reply:
Yes, thanks!